content/introduction/installation/docker.md

```diff
@@ -147,8 +122,165 @@ db_1  | TRACE: returning result row [INT64(0)]
 db_1  | TRACE: returning result row [INT64(1)]
 ```
 
+## Docker Image for Dolt SQL-Server
+
+We have [a Docker Image](https://hub.docker.com/r/dolthub/dolt-sql-server) for `dolt sql-server` creates a container 
```

Comment:
"Docker Image" should be "Docker image" : "Docker" is a proper noun; "image" is not. The section title on line 125 is fine of course since it's a heading and all main words are capitalized, but would be good to check the rest of the doc for this for consistency.